java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture4/tasks/_2/Number2.java

```diff
@@ -19,61 +19,87 @@ public static void main(String[] args) {
         binaryTree.delete(node, 13);
         binaryTree.delete(node, 10);
         binaryTree.order(node);
+        System.out.println("ROOT NODE: " + binaryTree.getRootNodeValue());
     }
 }
 
-class BinaryTree {
+record BinaryTree(Node rootNode) {
```

Comment:
records represents immutable classes; BinaryTree can change its state, so it's better to define it as a regular class